webcomponents/tool/src/main/frontend/packages/sakai-grader/src/sakai-gradable-data-mixin.js

```diff
@@ -11,7 +11,7 @@ export const gradableDataMixin = Base => class extends Base {
     return new Promise(resolve => {
 
       // Then, request the full set of data
-      const url = `/direct/assignment/gradable.json?gradableId=${gradableId}${submissionId ? `&submissionId=${submissionId}` : ""}`;
+      const url = `/direct/assignment/gradable.json?gradableId=${gradableId}${submissionId ? `&submissionId=${submissionId}` : ""}&mergeOverride=false`;
```

Comment:
gradable.json is only ever called from the grader code and you are always setting mergeOverride to false. So, can't you just update the entity provider logic to just not override with the individual student grade?

Reply:
Sure, assuming that Grader is the sole client of the getGradableForSite method of AssignmentEntityProvider. If that's confirmed, then I can refactor the code to remove this extra parameter.

Reply:
It definitely is. I wrote the endpoint just for the grader

Reply:
Ok, good to know.